core/domain/exp_jobs_one_off.py

```diff
@@ -33,14 +33,16 @@
 from core.domain import rights_manager
 from core.platform import models
 import feconf
+from pylatexenc import latex2text
```

Comment:
Normally, third-party stuff comes after our internal stuff -- see the mutagen example again. Does pylint require you to place this here? If so, that's a bit odd and you should talk to @Hudda.

Reply:
@Hudda I'm getting an `Imports are incorrectly sorted` error if I import this third-party lib after the internal imports. Could you PTAL? Thanks!